docs/docs/Get-Started/get-started-installation.md

```diff
@@ -3,170 +3,223 @@ title: Install Langflow
 slug: /get-started-installation
 ---
 
-Langflow can be installed in two ways:
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
-* As a [Python package](#install-langflow-locally)
-* As a [standalone desktop application](#install-langflow-desktop)
+Langflow can be installed in three ways:
 
-## Install Langflow locally
+* As a [Python package](#install-and-run-langflow-oss) with Langflow OSS
+* As a [standalone desktop application](#install-langflow-desktop) with Langflow Desktop
```

Comment:
This link doesn't work in the preview build.

Reply:
Fixed